src/store/reducers/posting-project-user/project/project.js

```diff
@@ -1,18 +1,79 @@
 import {
-    SET_PROJECT
-} from '../../../actions/posting-project-user/project/project'
+  SET_PROJECT,
+  ADD_NAME,
+  ADD_DESCRIPTION,
+  ADD_LOCATION,
+  ADD_DEADLINE,
+  ADD_QUESTION,
+  ADD_BENEFIT,
+  ADD_REQUIREMENT,
+  ADD_AVATAR,
+} from "../../../actions/posting-project-user/project/project";
 
 const initialState = {
-    projects: [],
+  projects: [],
+  name: null,
+  description: null,
+  location: null,
+  deadline: null,
+  question: null,
+  benefit: null,
+  requirement: null,
+  avatar: null,
```

Comment:
A good but not effective tackle
Remember that, each project, contains all those attributes (name, description, etc...)
**Please, look at the DFD and Data models described in the Miro file**